_translator-files/po/de/1-index.po

```diff
@@ -119,7 +120,7 @@ msgstr "Jamulus Dokumentation"
 #: ../wiki/en/misc/1-index.html:38
 #, no-wrap
 msgid ". Especially consider the "
-msgstr ". Vielleicht wird dein Problem auf der "
+msgstr ". Wirf einen Blick in die "
```

Comment:
Wirf einen Blick auf die

Probably?

Reply:
Latest version:
'Werfe einen Blick in unsere [Richtlinien für Beiträge](http://127.0.0.1:4000/de/wiki/Contribution) Wir freuen uns auf dich!'
OK?

Reply:
That's also ok...

Reply:
Alright

Reply:
Added the "."